src/main/java/seedu/address/logic/parser/ViewCompanyCommandParser.java

```diff
@@ -0,0 +1,51 @@
+package seedu.address.logic.parser;
+
+import static seedu.address.logic.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.logic.Messages.MESSAGE_MISSING_INDEX;
+import static seedu.address.logic.Messages.MESSAGE_UNKNOWN_COMMAND;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.ViewCompanyCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+
+/**
+ * Parses input arguments and creates a new ViewCompanyCommand object.
+ */
+public class ViewCompanyCommandParser implements Parser<ViewCompanyCommand> {
+
+    /**
+     * Parses the given {@code String} of arguments in the context of a ViewCompanyCommand
+     * and returns a ViewCompanyCommand object for execution.
+     *
+     * @throws ParseException if the user input does not conform to the expected format
```

Comment:
Capital is needed for the first character after the variable/exception. Add period after the sentence ends.

i.e
@throws ParseException If the user input does not conform to the expected format.